src/main/java/net/mcreator/ui/workspace/WorkspacePanel.java

```diff
@@ -358,21 +361,11 @@ public WorkspacePanel(final MCreator mcreator) {
 
 		modElementsBar.add(new JEmptyBox(7, 1));
 
-		JComponent isize = ComponentUtils.deriveFont(L10N.label("workspace.elements.list.icon_size"), 12);
-		isize.setToolTipText(L10N.t("workspace.elements.list.icon_size.tooltip"));

```

Comment:
Why was this section removed?

Reply:
Maybe an accident

Reply:
Please thoughtfully test PRs before submitting them